frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalette.test.tsx

```diff
@@ -190,7 +237,10 @@ describe('go to ERD with option select', () => {
 
     expect(activeTableNameDisplay).toBeEmptyDOMElement()
 
-    await user.click(within(dialog).getByRole('option', { name: 'follows' }))
+    const followsOption = within(dialog).getByRole('option', {
+      name: 'follows',
+    })
+    await user.dblClick(followsOption.firstChild as Element)
```

Comment:
[For the same reason with this](https://github.com/liam-hq/liam/pull/2127#discussion_r2160212315), it calls `await user.dblClick(followsOption.firstChild as Element)` click the child element; although `await user.dblClick(followsOption)` will cause the same result.